userspace/ksud/src/module.rs

```diff
@@ -182,21 +178,21 @@ pub fn exec_post_fs_data() -> Result<()> {
         }
         println!("exec {} post-fs-data.sh", path.display());
 
-        // pre_exec is unsafe!
+        let mut command = &mut Command::new(&post_fs_data); // faulty script which does not have a proper shebang will fail to be executed
```

Comment:
This may break compatibility with Magisk's module, please use `sh` to exec it.

Reply:
But this allows the script to set the shebang or using ELF binary. It would be a better choice to read the first few bytes to detect if there is a shebang.

Reply:
But the compatibility is also important, to use shebang or ELF, you can exec it in this script.

Reply:
I think that the new implemention makes both works now.